### PR TITLE
Add service request defaults.

### PR DESCRIPTION
**Pull request by Raynos**
This allows us to specify request defaults for a serviceName
in the actual router.

r: @rf @shannili

### CODE REVIEW

File: node/hyperbahn/service_proxy.js [hunk 1]
```diff
@@ -139,9 +140,15 @@ function createServiceChannel(serviceName) {
...
     var isExit = self.egressNodes.isExitFor(serviceName);
     var mode = isExit ? 'exit' : 'forward';
-    var svcchan = self.channel.makeSubChannel({
+
+    var options = {
         serviceName: serviceName
-    });
+    };
+    if (self.serviceReqDefaults[serviceName]) {
+        options.requestDefaults = self.serviceReqDefaults[serviceName];
```
Comment (ShanniLi) [inline node/hyperbahn/service_proxy.js]: where did you actually set self.serviceReqDefaults[serviceName]?

**Comment by rf** [comment]
:+1:

**Comment by ShanniLi** [comment]
:+1:

File: node/hyperbahn/service_proxy.js (hunk 1 above)
Comment (Raynos) [inline node/hyperbahn/service_proxy.js]: in autobahn itself.

**Pull request merged**